### DISCUSSION TITLE
How to get a TWITTER_TOKEN?

### DISCUSSION THREAD

**Discussion by MuYunyun** [lowlighter/metrics#136] [Q&A] [answered]
hi, where does `TWITTER_TOKEN` get in secrets.TWITTER_TOKEN? Is it in https://apps.twitter.com/app/new to apply?

**Comment by lowlighter** [comment, marked as answer]
Hello!

Yes, it seems you have the right link.

What you should have is something like below:
<img src="https://user-images.githubusercontent.com/22963968/106670800-cbed3480-65ad-11eb-9e15-a8f5a8574a30.png" width="600">

The token you must use is the **Bearer token** one 🙂

**Comment by MuYunyun** [comment]
all right. thanks.

**Comment by lowlighter** [comment]
Glad it helped 🙂
